packages/tables/src/Table.php

```diff
@@ -393,6 +394,14 @@ public function isSearchable(): bool
         return $this->getLivewire()->isTableSearchable();
     }
 
+    public function getSelectRecordsLayout(): string
+    {
+        /** @var TableComponent $livewire */
+        $livewire = $this->getLivewire();
+
+        return invade($livewire)->getTableSelectRecordsLayout() ?? SelectRecordsLayout::RowStart;
```

Comment:
Do we need the `invade` for a public method?

Reply:
Removed